homeassistant/core.py

```diff
@@ -1658,40 +1658,42 @@ def async_set(
 class Service:
     """Representation of a callable service."""
 
-    __slots__ = ["job", "schema", "domain", "service"]
+    __slots__ = ["job", "schema", "domain", "service", "supports_response"]
```

Comment:
`supports_response` -> I wonder if this should be an enum to distinguish between:
- no response (default)
- Optional response
- Response only

Reply:
Agree a Boolean feels limiting here though I'm not sure what the use will be for. Can you elaborate on how this will be used? Would we surface in the UI the type of service?